eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/TableHead/index.js

```diff
@@ -113,7 +113,7 @@ const TableHead = props => {
       <tr>
         <SortableTH
           sortColumn="title"
-          colWidth="18%"
+          colWidth="27%"
```

Comment:
Can we remove a few percent from here and add it to permissions, that columns looking a little tight.

Reply:
Added a bit to permission instead